NanoProd/crab/Run3_2022_postEE/SMHiggs.yaml

```
@@ -4,7 +4,7 @@ config:
     era: Run3_2022_postEE
```

```suggestion
    era: Run3_2022EE
```
here and in all other MC configs for 2022EE